docs/get-started/alerting.md

```diff
@@ -6,64 +6,161 @@
+Alerts are split into four key components: alert rules, contact points, notification policies, and silences.
 
-For example, you might configure an alert to identify and notify you when MongoDB is down.
+### Alert rules
 
-Provide a simplified framework for configuring complex alert rules. 
+Alert rules describe the circumstances under which you want to be alerted. The evaluation criteria that you define determine whether an alert will fire. An alert rule consists of one or more queries and expressions, a condition, the frequency of evaluation, and the duration over which the condition is met. For example, you might configure an alert to identify and notify you when MongoDB is down.
 
 PMM includes a set of default templates with common events and expressions for alerting. You can also create your own templates if you need custom expressions on which to base your alert rules.
 
-You can check the  alert templates available for your account under **Alerting > Alert rule templates** tab. PMM lists here the following types of templates:
+You can check the alert templates available for your account under **Alerting > Alert rule templates** tab. PMM lists here the following types of templates:
 
 1. Built-in templates, available out-of-the-box with PMM.
-2. Custom templates created or uploaded on the **Alerting page > Alert Templates** Tab. 
-3. Custom template files available in your  ``yaml srv/alerting/templates`` directory. PMM loads them during startup.
-
-### Silences
-Silences specify periods of time to suppress alerts and their associated notifications. During a silence, PMM continues to track metrics and trigger alerts but does not send notifications to the specified contact points. Once the silence expires alerts and notifications will resume.
-For example, you can create a silence to suppress trivial notifications during weekends.
+2. Custom templates created or uploaded on the **Alerting page > Alert Templates** tab.
+3. Custom template files available in your ``srv/alerting/templates`` directory. PMM loads them during startup.
 
 ### Contact points
-Contact points specify how PMM should deliver Grafana-managed alerts. When an alert fires, a notification is sent to the specified contact points. 
 
-Depending on the severity of an alert, you might want to send different alerts to different channels. For example, you can deliver common alerts via Slack channel, but send an email notification for potentially critical issues. 
+Contact points specify how PMM should deliver Grafana-managed alerts. When an alert fires, PMM sends a notification to the specified contact points.
+
+Depending on the severity of an alert, you might want to send different alerts to different channels. For example, you can deliver common alerts via Slack channel, but send an email notification for potentially critical issues.  
```

Comment:
"you might want to send different alerts" **(notifications)**. I have stated before that people use the term alert and notification interchangeably (and shouldnt)...and we are not making it clearer by 'muddying the waters' by saying (in the same sentence) about "delivering alerts by slack" but "but send an email notification"...so which is it? Lets be consistent in the same sentencethey are BOTH notifications, not alerts...I can live with Alert Notifications as well, but not just alerts